sycl/include/sycl/detail/pi.h

```diff
@@ -77,9 +77,11 @@
 // 12.22 Add piGetDeviceAndHostTimer to query device wall-clock timestamp
 // 12.23 Added new piextEnqueueDeviceGlobalVariableWrite and
 // piextEnqueueDeviceGlobalVariableRead functions.
+// 12.24 Added PI_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES and
```

Comment:
@kbenzie: please take this change to Unified Runtime

Reply:
Tracking in https://github.com/oneapi-src/unified-runtime/issues/399